src/app/campaign.service.ts

```diff
@@ -13,7 +13,7 @@ import { SelectedType } from './search.service';
 export class CampaignService {
   static perPage = 6;
 
-  private apiPath = '/campaigns/services/apexrest/v1.0';
+  private apiPath = '/campaigns/services/apexrest/v1.0' as const;
```

Comment:
This isn't related to the ticket, just a separate tiny quality of life improvement for devs.